src/components/Home.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable max-len */
```

Comment:
- [ ] I like your code, it's clean and classic :hugs: However, to further ensure compliance with industry rules and standards regarding coding styles and formatting :wink: and that is linters are meant to enforce, but this line is potentially avoiding compliance :smile: . Kindly remove this line and try to fix the lining issues. 
- This is what you will have to deal with daily when you finally get a placement with some company in the future. All companies have their rules to ensure uniformity and ease scalability for continuity :wrench: